godtools/App/Features/AppLanguage/Presentation/DownloadableLanguages/DownloadableLanguagesViewModel.swift

```diff
@@ -54,9 +56,45 @@ class DownloadableLanguagesViewModel: ObservableObject {
                 let downloadableLanguages = domainModel.downloadableLanguages
                 
                 self?.navTitle = interfaceStrings.navTitle
-                self?.downloadableLanguagesSearchResults = downloadableLanguages
+                self?.downloadableLanguages = downloadableLanguages
             }
             .store(in: &cancellables)
+        
+        Publishers.CombineLatest(
+            $downloadableLanguages.eraseToAnyPublisher(),
+            $activeDownloads.eraseToAnyPublisher()
+        )
+        .receive(on: DispatchQueue.main)
+        .sink(receiveValue: { [weak self] downloadableLanguages, activeDownloads in
+            
+            self?.updateDisplayedLanguages(from: downloadableLanguages, activeDownloads: activeDownloads)
+        })
+        .store(in: &cancellables)
+    }
+}
+
+// MARK: - Private
+
+extension DownloadableLanguagesViewModel {
+    
+    private func updateDisplayedLanguages(from downloadableLanguages: [DownloadableLanguageListItemDomainModel], activeDownloads: [BCP47LanguageIdentifier: LanguageDownloadStatusDomainModel]) {
+        
+        var updatedLanguages: [DownloadableLanguageListItemDomainModel] = Array()
+        
+        for downloadableLanguage in downloadableLanguages {
```

Comment:
My only concern with looping here is if we had a lot of downloadableLanguages you could see some UI slowdown here.  Would be good to do some performance tests.